torq/docker-compose.yml

```diff
@@ -0,0 +1,42 @@
+version: "3.7"
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: torq_web_1
+      APP_PORT: 7028
+  
+  web:
+    image: lncapital/torq:0.10.3@sha256:cacd93adc606f9e1e6740c9402a00caeb9490971b7715b8d1316a66a1af12749
+    user: "1000:1000"
+    restart: on-failure
+    stop_grace_period: 1m
+    volumes:
+      - ${APP_LIGHTNING_NODE_DATA_DIR}:/lnd:ro
+    command:
+      - --torq.port
+      - "7028"
+      - --db.host
+      - torq_db_1
+      - --db.password
+      - moneyprintergobrrr
+      - --torq.password
+      - ${APP_PASSWORD}
+      - --lnd.url
+      - ${APP_LIGHTNING_NODE_IP}:${APP_LIGHTNING_NODE_GRPC_PORT}
```

Comment:
During testing I had a small issue with my LND instance. Torq would get stuck in a reboot loop with this error:

```
2022/11/20 13:05:36 WARNING: [core] [Channel #1 SubChannel #2] grpc: addrConn.createTransport failed to connect to {
  "Addr": "10.21.21.9:10009",
  "ServerName": "10.21.21.9:10009",
  "Attributes": null,
  "BalancerAttributes": null,
  "Type": 0,
  "Metadata": null
}. Err: connection error: desc = "transport: error while dialing: dial tcp 10.21.21.9:10009: connect: no route to host"
{"level":"error","error":"Getting public key from node: Can't connect to node to verify public key, check all details including TLS Cert and Macaroon: cannot dial to lnd: connection error: desc = \"transport: error while dialing: dial tcp 10.21.21.9:10009: connect: no route to host\"","time":"2022-11-20T13:05:36Z","message":"Adding node specified in config to database"}
{"level":"fatal","error":"Adding node specified in config to database: Getting public key from node: Can't connect to node to verify public key, check all details including TLS Cert and Macaroon: cannot dial to lnd: connection error: desc = \"transport: error while dialing: dial tcp 10.21.21.9:10009: connect: no route to host\"","time":"2022-11-20T13:05:36Z"}
```

This is understandable, however, it would be nicer user experience, if the container remained up and displayed this error in the browser. I'm not going to block this PR on this - just a suggestion.

Reply:
Thanks for the feedback. We have recently done some work to retry the connection and to not prevent the webserver from coming up in this situation which should be a marginally better experience. Showing the current connection status (with any associated errors) is something we will be looking into very shortly. Much requested!